plugins/catalog-backend/src/next/database/DefaultProcessingDatabase.ts

```diff
@@ -121,6 +122,15 @@ export class DefaultProcessingDatabase implements ProcessingDatabase {
       this.deduplicateRelations(relationRows),
       BATCH_SIZE,
     );
+
+    await tx<DbFinalEntitiesRow>('final_entities')
```

Comment:
I realized we probably don't need this anymore now that we switched to a ticket rather than generation counter? Should be fine to insert with a merge on conflict in the beginning of the stitcher instead.